ambry-server/src/main/java/com/github/ambry/server/AmbryServerRequests.java

```diff
@@ -419,34 +429,68 @@ private AdminResponse handleBlobStoreControlRequest(DataInputStream requestStrea
    */
   private AdminResponse handleHealthCheckRequest(DataInputStream requestStream, AdminRequest adminRequest)
```

Comment:
Who sends a healthCheckRequest to ambry ?

Reply:
ambry-healthchecker will be the one sending requests to know about the server's health